### PR TITLE
fixes bug 1356571 - ignore version checking on certain products

### PR DESCRIPTION
**Pull request by peterbe**
cc @willkg @m8ttyB 

Another SeaMonkey thing. Thus I'm not going to scrutinize this problem too much. 

I do see SeaMonkey crashes coming in to stage. But none of the recent ones come from any of *active* SeaMonkey versions.

### CODE REVIEW

File: e2e-tests/tests/test_api.py [hunk 1]
```diff
@@ -8,6 +8,12 @@
 import pytest
 
 
+# Because there are so few crashes for these product, instead of
+# looking for crashes from the *active* versions, we'll for any
+# any recent crashes for these.
```
Comment (willkg) [inline e2e-tests/tests/test_api.py]: This last clause has two "any" and is missing a word. Maybe it should be "we'll look for any recent crashes"?

File: e2e-tests/tests/test_api.py [hunk 2]
```diff
@@ -26,6 +32,9 @@ def test_public_api_navigation(self, base_url):
         # for a single crash for that product
         product_to_uuid = {}
         for product, versions in product_versions.items():
+            if product == ANY_VERSION_PRODUCTS:
```
Comment (willkg) [inline e2e-tests/tests/test_api.py]: Should this be:

```
if product in ANY_VERSION_PRODUCTS:
```

?

Comment (peterbe) [inline e2e-tests/tests/test_api.py]: Wow. Yeah. Bad refactoring when I cleaned up.

**Comment by peterbe** [comment]
Nits addressed.

**Pull request merged**